templates/README

```diff
@@ -1 +1,60 @@
-This contains some basic description about the BIDS dataset.
+### Introduction
+The README is usually the starting point for researchers using your data
+and serves as a guidepost for users of your data. A clear and informative
+README makes your data much more usable.
+
+Overview
+
+A paragraph giving an overview of the experiment. This should include the 
+goals or purpose and a discussion about how the experiment tries to achieve
+these goals.
+
+A brief overview of the tasks in the experiment.
+
+A brief discussion of condition variables (sometimes called contrasts
+or independent variables) that were varied across the experiment.
+
+A brief discussion of the response variables (sometimes called the
+dependent variables) that were measured and or calculated to assess
+the effects of varying the condition variables. This might also include
+questionnaires administered to assess behavioral aspects of the experiment.
+
+A brief discussion of the control variables --- that is what aspects 
+were explicitly controlled in this experiment. The control variables might 
+include subject pool, environmental conditions, set up, or other things 
+that were explicitly controlled.
+
+More details on all of this should be included in the Methods section.
+
+
+Methods  
+
+Subjects: A brief sentence about the subject pool in this experiment.
+ 
+Apparatus: A summary of the equipment and environment setup for the
+experiment. For example, was the experiment performed in a shielded room
+with the subject seated in a fixed position.
+
+Initial setup: A summary of what setup was performed when a subject arrived.
+
+Task organization: How the tasks were organized for a session.
+This is particularly important because BIDS datasets usually have task data
+separated into different files.) What was the order that tasks were performed? 
+Was task order counter-balanced?  What other activities were interspersed 
+because the tasks?  
+
+Task details: As much detail as possible about the task and the
+events that were recorded. 
+
+Additional data acquired: A brief indication of data other than the
+imaging data that was acquired as part of this experiment. In addition
+to data from other modalities and behavioral data, this might include
+questionnaires and surveys, swabs, and clinical information. Indicate
+the availability of this data.
+
+Experimental Location: This should include the geographical 
+location and facility.
+
+Notes: Any additional information or pointers to information that
+might be helpful to users of the dataset.
+
```

Comment:
Maybe add some information related to "using" the dataset: I am specifically thinking of those datasets that are datalad curated.

A link to the relevant section of the datalad README might help.

http://handbook.datalad.org/en/latest/basics/101-180-FAQ.html#how-can-i-help-others-get-started-with-a-shared-dataset

Reply:
I'm not sure if this would be relevant within the README (but might be useful to mention here as well?) but just adding to Remi's suggestion

- if the dataset requires a data use agreement, then linking information regarding that 
- if the dataset includes a `derivatives` directory, then a few lines about what sort of pre-processing the raw (Nifti) data went through, tools used in the process, etc 
for example, if the data was processed through fmriprep, then details of the version of fmriprep used.

Reply:
hey @Arshitha

I think part of that info for derivatives is in the `dataset_description.json`: so I think that maybe we want to avoid redundancies in the README. But for anything that is not in that json then, it can go in the README.

Good idea about the DUA. :-)